tuned/plugins/plugin_scheduler.py

```diff
@@ -318,15 +318,15 @@ def _tune_process(self, pid, cmd, sched, prio, affinity):
 	def _convert_sched_params(self, str_scheduler, str_priority):
 		scheduler = self._dict_schedcfg2num.get(str_scheduler)
 		if scheduler is None and str_scheduler != "*":
-			log.error("Invalid scheduler: %s. Scheduler and priority will be ignored."
-					% str_scheduler)
+			log.warn("Invalid scheduler: %s. Scheduler and priority will be ignored."
+					 % str_scheduler)
```

Comment:
I'd say error is the correct log level here, as it can only happen due to misconfiguration. All the valid scheduler names are defined in `_dict_schedcfg2num`. If anything else is used in the profile, it's misconfiguration.